docs/get-started/lotus/send-and-receive-fil.md

```diff
@@ -18,58 +18,58 @@ In order to receive and send FIL with Lotus you will need to have [installed and
 lotus wallet new bls
 ```
 
-This will create a new wallet and print its Filecoin address. You can distinguish mainnet from testnet addresses because they start with `f` for mainnet and with `t` for testnets.
+This will create a new wallet and print its Filecoin address. You can distinguish mainnet from testnet addresses because they start with `f` for mainnet and `t` for testnets.
```

Comment:
```suggestion
This will create a new address and print it. You can distinguish mainnet from testnet addresses because they start with `f` for mainnet and `t` for testnets.
```